code/scripts/tex_build.sh

```diff
@@ -24,17 +24,17 @@ fi
 
 GEN_NAME_SUFFIX=_SRS
 
-cd "$BUILD_FOLDER$EDIR"/SRS/PDF
+cd "$BUILD_FOLDER$EDIR"/SRS/PDF || exit
 "$MAKE" TEXFLAGS="-interaction=$IMODE --shell-escape" BIBTEXFLAGS="$BIFLAGS"
 RET=$?
 
 if [ "$SUMMARIZE_TEX" = "yes" ]; then
-  echo "\n\n\033[0;33m$EDIR TeX Summary\033[0m:"
+  printf "\n\n\033[0;33m%s TeX Summary\033[0m:" "$EDIR"
   if [ "$RET" -eq 0 ]; then
     # Approximate error gathering from TeX logs.
-    cat "$EDIR$GEN_NAME_SUFFIX".log | grep -E "erfull|Warning"
-    cat "$EDIR$GEN_NAME_SUFFIX".blg | grep -B3 -E "Error"
-    BIBERRS=$(cat "$EDIR$GEN_NAME_SUFFIX".blg | grep -E "Error" | wc -l)
+    grep -E "erfull|Warning" < "$EDIR$GEN_NAME_SUFFIX".log
+    grep -B3 -E "Error" < "$EDIR$GEN_NAME_SUFFIX".blg
```

Comment:
I think we can also remove the `<` on these lines.

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3664/commits/b1f11dde00fc5e9e77c04442f4b29ce1395404d4.